e2e/runner/bitcoin.go

```diff
@@ -286,6 +296,57 @@ func (r *E2ERunner) SendToTSSFromDeployerWithMemo(
 	return txid, nil
 }
 
+func (r *E2ERunner) InscribeToTSSFromDeployerWithMemo(
+	amount float64,
+	inputUTXOs []btcjson.ListUnspentResult,
+	memo []byte,
+) (*chainhash.Hash, error) {
+	builder := InscriptionBuilder{sidecarUrl: "http://bitcoin-node-sidecar:8000", client: http.Client{}}
+
+	address, err := builder.GenerateCommitAddress(memo)
+	if err != nil {
+		return nil, err
+	}
+	r.Logger.Print("received inscription commit address %s", address)
+
+	receiver, err := chains.DecodeBtcAddress(address, r.GetBitcoinChainID())
+	if err != nil {
```

Comment:
You could simply replace `if err != nil {...}`with `require.NoError(r, err)` to make the func compact.

Reply:
done